src/main/java/org/sagebionetworks/web/client/presenter/EmailInvitationPresenter.java

```diff
@@ -148,18 +146,19 @@ public void onFailure(Throwable t) {
             view.hideLoading();
             if (t instanceof ForbiddenException) {
               // SWC-4721: fix message in the case where membership invitation email is not associated to the
-              // currently logged in user
+              // currently logged-in user
               view.showErrorMessage(
                 "This invitation was sent to an email address not associated to the current user. \"" +
                 membershipInvitation.getInviteeEmail() +
-                "\" Please add this email to your Synapse account under \"Settings\", or log in with the correct Synapse account before accepting the invitation."
+                "\" Please add this email to your Synapse account under \"Edit Profile\", or log in with the correct Synapse account before accepting the invitation."
               );
-              // SWC-4741: invitation not associated to the current user, send user to the Settings page to add
-              // the new email address
+              // SWC-4741: invitation not associated to the current user, send user to the Profile page, where they can
+              // click 'Edit Profile', then 'Change email' to get to OneSage and add the new email address
+              // NOTE: We don't send the user directly to OneSage so that they have a chance to read the message first
               placeChanger.goTo(
                 new Profile(
                   authController.getCurrentUserPrincipalId(),
-                  ProfileArea.SETTINGS
+                  ProfileArea.PROFILE
```

Comment:
Keep showing an error message for the edge case where you try to join a team using an emailed link, but the email address is not tied to your account. 

From the profile page, the user can follow a few steps to change their email address, so this should be good enough for this uncommon scenario.